kafka.go

```diff
@@ -149,6 +148,11 @@ func newKafkaClient(brokers []string, prefix string, config ...*BrokerConfig) (*
 	if len(config) > 0 {
 		err = setConfig(writerConfig, readerConfig, config[0])
 		strictValidation = config[0].StrictValidation
+		if config[0].Balancer != nil {
+			writerConfig.Balancer = config[0].Balancer
+		} else {
+			writerConfig.Balancer = &kafka.RoundRobin{}
+		}
```

Comment:
minor feedback:
`writerConfig.Balancer = &kafka.RoundRobin{}
if config[0].Balancer != nil {
	writerConfig.Balancer = config[0].Balancer
}`

Reply:
with this, it will take 3 processes if the balancer is provided. using if else will only take 2 processes.

Reply:
if the default balancer is round-robin, don't need to set Else statement

Reply:
okay will remove it from this lib